src/api/handlers/twilio/postReply.js

```diff
@@ -1,15 +1,157 @@
+import moment from "moment";
+import { db } from "../../../../knex";
 import TemplatesModel from "../../../models/templates";
 import { getMessageResponse } from "../../../services/twilioClient";
 
-export default (req, res) => {
+const convertReply = (reply) => {
+  const languageConversions = {
+    yes: ["نعم", "አዎ", "haa", "Evet", "si", "yes"],
+    // In spanish and english no is the same
+    no: ["لا", "አይ", "maya", "Hayır", "no"],
+    interpreter: [
+      "مترجم",
+      "አስተርጓሚ",
+      "turjubaan",
+      "çevirmen",
+      "interprete",
+      "interpreter",
+    ],
+  };
+  let convertedReply;
 
-  const language = "English";
+  for (let key of Object.keys(languageConversions)) {
+    if (languageConversions[key].includes(reply)) {
+      convertedReply = key;
+      break;
+    }
+  }
 
-  const replyText = TemplatesModel.generateReply(language);
+  return convertedReply;
+};
+
+export async function getAppointments(patientPhoneNumber) {
+  return await db("appointments")
```

Comment:
I think this should return the `DB` promise without `async/await`, as it's already handled below at line `52`

Reply:
Good idea, this function is still a work in progress. Might not be needed.